assets/scripts/app.js

```diff
@@ -20,3 +20,269 @@ const closeMenuHandler = () => {
+    modalTagMobile: ['Ruby on Rails', 'Css', 'Javascript'],
+    img: '../assets/images/modal-lg.png',
+    seeLive: 'http://ansderly.me/portfolio/',
+    seeSource: 'https://github.com/hansderly',
+    description: `Lorem Ipsum is simply dummy text of the printing and typesetting
+                            industry. Lorem Ipsum has
+                            been
+                            the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                            galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the
+                            printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text
+                            ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s
+                            with the relea
+`,
+  },
+  {
+    id: 5,
+    title: 'Multi-Post Stories Gain+Glory',
+    tags: ['Ruby', 'Css', 'JavaScript', 'Html'],
+    modalTagDesktop: ['Codekit', 'Github', 'Javascript', 'Bootstrap', 'Terminal', 'Codepen'],
+    modalTagMobile: ['Ruby on Rails', 'Css', 'Javascript'],
+    img: '../assets/images/modal-lg.png',
+    seeLive: 'http://ansderly.me/portfolio/',
+    seeSource: 'https://github.com/hansderly',
+    description: `Lorem Ipsum is simply dummy text of the printing and typesetting
+                            industry. Lorem Ipsum has
+                            been
+                            the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                            galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the
+                            printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text
+                            ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s
+                            with the relea
+`,
+  },
+];
+
+const addWorkCard = () => {
+  const recentWorkGridContainer = document.querySelector(
+    '.recent-work-grid-container',
+  );
+  recentWorks.map((work) => {
+    const card = document.createElement('div');
+    card.classList.add('single-card-container');
+    card.innerHTML = `
+      <div class="single-card">
+          <div class="img-single-work"></div>
+          <div class="infos-container">
+              <header class="name-single-work">
+                  <h3>${work.title}</h3>
+              </header>
+              <div class="stack-single-work-container">
+                  <ul class="stack-single-container">
+                    <li class="stack"><a href="#">${work.tags[0]}</a></li>
+                    <li class="stack"><a href="#">${work.tags[1]}</a></li>
+                    <li class="stack"><a href="#">${work.tags[2]}</a></li>
+                    <li class="stack"><a href="#">${work.tags[3]}</a></li>
```

Comment:
- [ ] - [ OPTIONAL ]This piece of code works fine since it renders the tags array element, but I think it is not clean and flexible in the future, imagine you need to add a project with 50 technologies may be, with this logic you will be calling tags[0] up to tags[49], which is tiring, instead, since the tags in an array, you can simply `map` over it and return the items easily 
``` 
works.tags.map((tech) => {
 <li>tech</li>
})
```